pkg/zcmd/zpool/get/builder.go

```diff
@@ -153,35 +152,35 @@ func (p *PoolGProperty) Build() (*PoolGProperty, error) {
 	p = p.Validate()
 	p.appendCommand(&c, bin.ZPOOL)
 
-	p.appendCommand(&c, fmt.Sprintf(" %s ", Operation))
+	p.appendCommand(&c, " "+Operation+" ")
 
 	if IsScriptedModeSet()(p) {
-		p.appendCommand(&c, fmt.Sprintf(" -H "))
+		p.appendCommand(&c, " -H ")
 	}
 
 	if IsParsableModeSet()(p) {
-		p.appendCommand(&c, fmt.Sprintf(" -p "))
+		p.appendCommand(&c, " -p ")
 	}
 
 	if IsFieldListSet()(p) {
-		p.appendCommand(&c, fmt.Sprintf(" -o "))
+		p.appendCommand(&c, " -o ")
 		for _, f := range p.FieldList {
-			p.appendCommand(&c, fmt.Sprintf("%s,", f))
+			p.appendCommand(&c, " "+f+" ")
```

Comment:
This should be something like below... It is not expecting for space
```suggestion
			p.appendCommand(&c, f+", ")
```
Above is equivalent to `p.appendCommand(&c, f + ",")`